system/netdata.service.in

```diff
@@ -1,15 +1,56 @@
+# netdata systemd target
+
 [Unit]
-Description=Real time performance monitoring
+Description=netdata - Real-time performance monitoring
+#Documentation=man:netdata
+#Documentation=file:///usr/share/doc/netdata/html/index.html
+Documentation=https://github.com/firehol/netdata/wiki
 After=network.target httpd.service squid.service nfs-server.service mysqld.service named.service postfix.service
+Wants=network-online.target
+ConditionPathExists=@sysconfig_POST@/netdata/netdata.conf
```

Comment:
I am not sure if we want this. In my opinion netdata should start even without configuration. @ktsaou ?